app/dashboard/models.py

```diff
@@ -65,18 +70,20 @@ class Meta:
     project_length = models.CharField(max_length=50, choices=PROJECT_LENGTHS)
     experience_level = models.CharField(max_length=50, choices=EXPERIENCE_LEVELS)
     github_url = models.URLField()
+    github_comments = models.IntegerField(default=0)
     bounty_owner_address = models.CharField(max_length=50)
     bounty_owner_email = models.CharField(max_length=255, null=True)
     bounty_owner_github_username = models.CharField(max_length=255, null=True)
     claimeee_address = models.CharField(max_length=50)
     claimee_email = models.CharField(max_length=255, null=True)
     claimee_github_username = models.CharField(max_length=255, null=True)
-    is_open = models.BooleanField()
+    is_open = models.BooleanField(help_text='Whether the bounty has been filled')
     expires_date = models.DateTimeField()
     raw_data = JSONField()
     metadata = JSONField(default={})
     claimee_metadata = JSONField(default={})
-    current_bounty = models.BooleanField(default=False) # whether this bounty is the most current revision one or not
+    current_bounty = models.BooleanField(default=False,
+                                         help_text='whether this bounty is the most current revision one or not')
```

Comment:
Just a minor nitpick...  Should we move forward with a standard of capitalizing the first letter in `help_text` or keep it all lowercase?